lms/djangoapps/courseware/plugins.py

```diff
@@ -267,3 +267,47 @@ def get_allowed_operations(cls, course_key: CourseKey, user: Optional[User] = No
     @staticmethod
     def legacy_link(course_key: CourseKey):
         return urls.reverse('tabs_handler', kwargs={'course_key_string': course_key})
+
+
+class OpenResponsesApp(CourseApp):
+    """
+    Course App config for ORA app.
+    """
+
+    app_id = "open_responses"
+    name = _("Open Responses")
+    description = _("Turn on flexible peer grading for all open responses.")
```

Comment:
This description should probably also change to something like "Set Course-level settings for Open Response Assessments" or something like that.